content/en/observability_pipelines/live_capture.md

```diff
@@ -39,5 +45,6 @@ Specifically, the following information is shown:
 
 {{< partial name="whats-next/whats-next.html" >}}
 
-[1]: https://app.datadoghq.com/observability-pipelines
-
+[1]: /rbac/permissions/#observability-pipelines
```

Comment:
Hey May! I noticed that this link wasn't working when I clicked on it. Is this what it's supposed to be?

```suggestion
[1]: /account_management/rbac/permissions/#observability-pipelines
```